ProjectSettings/QualitySettings.asset

```diff
@@ -187,7 +187,7 @@ QualitySettings:
     Web: 5
     WebGL: 3
     WiiU: 5
-    Windows Store Apps: 0
+    Windows Store Apps: 5
```

Comment:
What's this value mapping to?

Reply:
Changing the quality from Fastest to Fantastic. Should be reverted as well

Reply:
I've no idea why these keep getting changed.  Will revert back to those in the dev working branch in a new PR.

Reply:
Resolved in #1648